src/software/networking/udp_sender.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include <boost/asio.hpp>
+#include <boost/bind.hpp>
+#include <string>
+
+class UdpSender
```

Comment:
why did you change this from a ProtoUdp sender to just a UdpSender

Reply:
It's to have a more general class for sending UDP. I needed it so I can send JSON strings to plotjuggler. The UDP class hierarchy now follow the structure of the UnixSender more closely as well.